KMMYCharts/src/commonMain/kotlin/co/yml/kmm/charts/ui/linechart/LineChart.kt

```diff
@@ -101,7 +101,7 @@ internal fun LineChart(modifier: Modifier, lineChartData: LineChartData) {
                         xMax,
                         xMin,
                         xOffset,
-                        paddingRight.toPx(),
+                        paddingRight.toSp().toPx(),
```

Comment:
@codeangi Do we need sp conversion here ?

Reply:
No @dkk009 its a mistake, i've updated